library/std/convert.qs

```diff
@@ -16,6 +16,34 @@ namespace Microsoft.Quantum.Convert {
         body intrinsic;
     }
 
+    /// # Summary
+    /// Converts a `Result` type to a `Bool` type, where `One` is mapped to
+    /// `true` and `Zero` is mapped to `false`.
+    ///
+    /// # Input
+    /// ## input
+    /// `Result` to be converted.
+    ///
+    /// # Output
+    /// A `Bool` representing the `input`.
+    function ResultAsBool(input : Result) : Bool {
+        return input == One;
```

Comment:
```suggestion
        input == One
```
We prefer the code to be expression-based. Therefore, there's no need for 'return' in the last statement., especially in short functions like this.

Reply:
thanks for explaining this - make sense to me. I saw this in places, but wasn't sure especially as the old compiler did not support this